put vale in default location to have tools 'just work' (#29913)
quarkusio/quarkus: docs/src/main/asciidoc/doc-contribute-docs-howto.adoc

=== Use the Vale CLI

If you install the https://vale.sh/docs/vale-cli/installation/[install the Vale CLI], you can run the checks from the `docs` directory:

[source,shell]
----
cd docs
vale --minAlertLevel=warning src/main/asciidoc
----

> If you want to run it from another directory use `--config=/path/to/quarks/docs/.vale.ini`.

Reviewer: ```suggestion
If you want to run it from another directory use `--config=/path/to/quarkus/docs/.vale.ini`.
```

See the https://vale.sh/manual/[Vale CLI Manual] for details.

=== Vale IDE plugins

https://vale.sh/docs/integrations/guide/[Vale IDE integrations] rely on installation the Vale CLI.

> Each has it's own configuration requirements. The Visual Studio Code IDE extension, for example, requires you to tell where valeCLI is installed in workspace settings:

Reviewer: ```suggestion
Each has it's own configuration requirements. The Visual Studio Code IDE extension, for example, requires you to tell where the vale CLI is installed in workspace settings:
```

[source,json]
----
"vale.valeCLI.path": "/path/to/vale"
----

> The plugin will pick up the `.vale.ini` settings automatically relatively to the documentation files.

Reviewer: ```suggestion
The plugin will discover the `.vale.ini` configuration automatically.
```
